Library/Homebrew/cmd/bottle.rb

```diff
@@ -84,10 +87,15 @@ def print_filename(string, filename)
           result = true
 
           if ARGV.verbose?
+            strings_matches += 1
+            next if strings_matches > MAXIMUM_STRING_MATCHES
             print_filename string, file
             puts " #{Tty.gray}-->#{Tty.reset} match '#{match}' at offset #{Tty.em}0x#{offset}#{Tty.reset}"
           end
         end
+        if strings_matches > MAXIMUM_STRING_MATCHES
+          puts "Only the first #{MAXIMUM_STRING_MATCHES} matches were output"
+        end
       end
```

Comment:
Ok, but I still find it's weird to have `if strings_matches > MAXIMUM_STRING_MATCHES` outside of `if ARGV.verbose?`. How about:
```ruby
text_matches = []
Utils.popen_read("strings", "-t", "x", "-", file.to_s) do |io|
 ...
 text_matches << [match, offset]
end
if text_matches.any? && ARGV.verbose?
  print_filename string, file
  text_matches.first(MAXIMUM_STRING_MATCHES).each do |match, offset|
    puts " #{Tty.gray}-->#{Tty.reset} match '#{match}' at offset #{Tty.em}0x#{offset}#{Tty.reset}"
  end
  if text_matches.size > MAXIMUM_STRING_MATCHES
    puts "Only the first #{MAXIMUM_STRING_MATCHES} matches were output"
  end
end
```